packages/nuxt3/src/pages/macros.ts

```diff
@@ -13,6 +13,7 @@ export const TransformMacroPlugin = createUnplugin((options: TransformMacroPlugi
     name: 'nuxt:pages-macros-transform',
     enforce: 'post',
     transformInclude (id) {
+      if (!id) { return }
```

Comment:
Should we also exclude '\x00' for virtual assets?

```suggestion
      if (!id || id.startsWith('\x00')) { return }
```